packages/dev/core/src/XR/webXRCamera.ts

```diff
@@ -26,21 +26,12 @@ export class WebXRCamera extends FreeCamera {
      */
     public onXRCameraInitializedObservable = new Observable<WebXRCamera>();
 
-    /**
-     * Observable raised before camera teleportation
-     */
-    public onBeforeCameraTeleport = new Observable<Vector3>();

```

Comment:
This is a breaking change, I do not think we can do it ? cc @RaananW

Reply:
@sebavan, maybe I misunderstood what @RaananW said. I added the rotation observables to the feature but also moved the teleport observables to the feature.

I am totally ok in leaving the teleport observables in the camera I just feel they make more sense together.

Reply:
Yup my guess is the new ones should be on the feature but the old should unfortunately where they are fo back compat. but let s wait on @RaananW  review :-)

Reply:
@sebavan I just reverted the breaking changes. Should be fine now. Thanks for taking time to review it.